images/linux/toolsets/toolset-1804.json

```diff
@@ -42,12 +42,11 @@
             "arch": "x64",
             "platform" : "linux",
             "versions": [
-                "1.15.*",
                 "1.16.*",
                 "1.17.*",
                 "1.18.*"
             ],
-            "default": "1.15.*"
+            "default": "1.17.*"
```

Comment:
I think we should update default to 1.18.
Updating from 1.15 to 1.17 will be painful anyway so it is worth to update to 1.18 as actual stable version right now.
1.17 will reach EOL on 2022-08-16

Reply:
We run Kusto queries and seen that 1.17 is the most used, currently. Also, we support three latest versions.
What do you think?

Reply:
I think 1.17 is the most used because 1.18 was released recently and not all customers was able to switch to it yet.
That is probably up to you but keep on mind that if you update default to 1.17 for now, you will need to update to 1.18 one more time in 4-5 months.  It is a bit easier to bump it once.
I am not really sure how much breaking changes between 1.17 and 1.18 versions and how many customers will be affected. But I guess updating from 1.15 to 1.17 will be very painful anyway.

Reply:
we have announced 1.15 -> 1.17 switch already